tokens/tori.fi/semantic.yml

```diff
@@ -168,12 +168,12 @@ s:
       inverted:
         _: white
         subtle: gray-50
-      link: petroleum-600
+      link: blueberry-600
       disabled: gray-300
       negative: red-600
       positive: green-600
       notification: white
 
-    focused: petroleum-400
+    focused: blueberry-400
```

Comment:
Do we know if the focused color is the same in Figma (or wherever it needs to be :D ) so we don't need to update it later if it's not? I can't see any focused color in the [Taxonomy spreadsheet](https://docs.google.com/spreadsheets/d/1Q-Tr_dwJVfxgh3527IFjXQKqtnPPM42QeIr-M6q-zu8/edit#gid=1864511726).

Reply:
It looks like in the[ Figma overview ](https://www.figma.com/file/VcPPjSBp0sNMd5OuH1zVlJ/Tori-Colour-Concept?type=design&node-id=161-38448&mode=design&t=CFhHYoBqxsD62YU5-0)the outline might actually be `blue-300`.

Reply:
Changed to `blue-300` 👍